src/events/types/weather/LightningStrikeEvent.ts

```diff
@@ -0,0 +1,53 @@
+import { Cancellable } from "../../Cancellable";
+import { LightningStrikeEventCause } from "./LightningStrikeEventCause";
+import { WeatherEvent } from "./WeatherEvent";
+
+/**
+ * Stores data for lightning striking
+ */
+export class LightningStrikeEvent extends WeatherEvent implements Cancellable {
+    
+    public static getBukkitClasspath(): string {
+        return 'org.bukkit.event.weather.LightningStrikeEvent';
+    }
+   
+    /**
+    * Gets the cause of this lightning strike
+    * @returns strike cause
+    */
+    public getCause():LightningStrikeEventCause {
+        return this.toJava().getCause();
+    }
+
+    /**
+    * Sets the cancellation state of this event. A cancelled event will not be executed in
+    * the sever, but will still pass to other plugins
+    * @param cancel 
+    */
+    public setCancelled(cancel: boolean): void {
+        this.toJava().setCancelled(cancel);
+    }
+    
+    /**
+    * Gets the cancellation state of this event.A cancelled event will not be executed in the server,
+    * but will still pass to other plugins
+    * @returns true if this event is cancelled
+    */
+    public isCancelled(): boolean {
+      return this.toJava().isCancelled();
+    }
+
+    
+    /*
+    * Gets the bolt which is striking the earth
+    * @returns 
+    */
+        
+    /*
+    ---------- LightningStrike entity needs to be added -----------
+
+    public getLightning():LightningStrike {
```

Comment:
You can just return an Entity for now. 

Although as the Entity type has only one extra method, I'd prefer you go ahead and make the Entity class, don't have to though.